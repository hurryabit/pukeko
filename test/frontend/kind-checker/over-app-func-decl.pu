bad :: Int Int
bad = 0
