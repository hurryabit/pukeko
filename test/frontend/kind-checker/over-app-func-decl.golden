test/frontend/kind-checker/over-app-func-decl.pu:1:1: cannot unify kinds * and * -> *
